src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -6081,7 +6081,25 @@ && isJavadocVMInitError( output ) )
                 msg.append( exitCode );
                 if ( StringUtils.isNotEmpty( err.getOutput() ) )
                 {
-                    msg.append( " - " ).append( err.getOutput() );
+                    // parse stderr, log informational output, add all other to exception message
+                    List<String> nonInfoLines = new ArrayList<>();
+                    for ( String str : err.getOutput().split( "\\R" ) )
+                    {
+                        if ( isInfoOutput( str ) )
+                        {
+                            getLog().debug( str );
```

Comment:
Does it make sense to send info to debug?

Reply:
The method was named `isInfoOutput` in the sense of: is informational (e.g. not error) output of the Javadoc tool.
The mojo logs at debug and info levels in various places. It's a matter of taste whether the output identified by `isInfoOutput` should be logged at debug or info level. I am open to whichever. The naming of the method lets one expect that it should be logged as `info`, so I understand why you would ask the question.

Reply:
If it is inconsistent in our code, I'd like to ask to file a JIRA issue about this not to be forgotten.

Reply:
No, the existing code is just fine in my opinion.
The informational output of the tool is very verbose. I would therefore log at debug level only.